packages/xrpl/src/Wallet/rfc1751.ts

```diff
@@ -176,15 +178,34 @@ function bufferToArray(buf: Buffer): number[] {
  * @param buf - A 128-bit (16 byte) buffer
  * @returns A buffer containing the same data with reversed endianness
  */
-function swap128(buf: Buffer): Buffer {
-  // Interprets buffer as an array of (two, in this case) 64-bit numbers and swaps byte order in-place.
-  const reversedBytes = buf.swap64()
-
-  // Swap the two 64-bit numbers since our buffer is 128 bits.
-  return Buffer.concat(
-    [reversedBytes.slice(8, 16), reversedBytes.slice(0, 8)],
-    16,
-  )
+function swap128(buf: Uint8Array): Uint8Array {
+  // // Interprets buffer as an array of (two, in this case) 64-bit numbers and swaps byte order in-place.
+  const reversedBytes = swap64(buf)
```

Comment:
This puts 128 bytes into a 64 byte swap, I wouldn't expect this to work.

Reply:
Resolved by 2d66c38a06eb47f13102054958b599bace783b9d